### PR TITLE
Move ME byte conversion method.

### PR DESCRIPTION
**Pull request by Doug-Murphy**
Resolves CR #32 .

### CODE REVIEW

File: utils [hunk 1]
```diff
@@ -85,3 +85,22 @@ function rednetLookupWithRetry(protocol, host)
...
+  elseif byteLength > 6 and byteLength <= 9 then
+    postfix = "MB"
+    bytes = bytes / 1000000
+  elseif byteLength > 9 and byteLength <= 12 then
+    postfix = "GB"
+    bytes = bytes / 1000000000
+  end
+
+  return round(bytes, 1) .. postfix
+end
```
Comment (Joshua-Brent-Collins) [inline utils]: newline please

File: glassesHud [hunk 2]
```diff
@@ -54,11 +35,11 @@ while true do
...
-    ["ME Bytes"] = convertMEBytes(meStatus.usedBytes) .. "/" .. convertMEBytes(meStatus.totalBytes) .. " @ " .. meStatus.Updated,
+    ["ME Bytes"] = utils.convertMEBytes(meStatus.usedBytes) .. "/" .. utils.convertMEBytes(meStatus.totalBytes) .. " @ " .. meStatus.Updated,
     ["Blutonium"] = utils.formatInt(reactor["Blutonium Ingot"]) .. " @ " .. reactor["Updated"],
     ["Yellorium"] = utils.formatInt(reactor["Yellorium Ingot"]) .. " @ " .. reactor["Updated"]
   }
 
   display(displayText)
   sleep(25)
-end
+end
```
Comment (Joshua-Brent-Collins) [inline glassesHud]: also newline

File: utils (hunk 1 above)
Comment (Doug-Murphy) [reply utils]: Added.

File: glassesHud (hunk 2 above)
Comment (Doug-Murphy) [reply glassesHud]: Added.

**Pull request merged**